### PR TITLE
Fix for unstdlib

### PR DESCRIPTION
**Pull request by kennedyshead**
#10

### CODE REVIEW

**Comment by bbenne10** [comment]
Please recommit with a more descriptive name ("add unstdlib to setup.py" or something similar) and close the issue on a deeper line. I'll have to codify this is a contributing document or something soon.

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bbenne10) [inline .idea/vcs.xml:1]: This needs to be rebased out.

**Review by bbenne10 — COMMENTED** [review]
Once the `.idea` files go away, this looks fine.

**Comment by kennedyshead** [comment]
I'll do a new PR ;)

**Comment by bbenne10** [comment]
You can just update this branch and rebase out the file, but however you'd like to proceed is fine

**Pull request closed without merging**